storageClient/api.go

```diff
@@ -50,6 +50,19 @@ type DescribeGroupRequest struct {
 	GroupID uuid.UUID `json:"group_id"`
 }
 
+// ListGroupsRequest wraps values used to Look up Groups a client is associated with
```

Comment:
`look up groups for the client's account by default or optionally filter by parameters such as client ID`